CodeEditor/Sources/CodeEditor/UXCodeTextView.swift

```diff
@@ -20,7 +20,6 @@ typealias UXTextView          = UITextView
 typealias UXTextViewDelegate  = UITextViewDelegate
 #endif
 
-// swiftlint:disable type_body_length
 /**
  * Subclass of NSTextView/UITextView which adds some code editing features to
```

Comment:
For me, this is still needed:

<img width="1156" alt="screenshot-2023-01-22_000932" src="https://user-images.githubusercontent.com/9006596/213944048-6b6d0253-fc9d-41e4-8365-83fc7de1b626.png">